lms/envs/common.py

```diff
@@ -3217,6 +3217,10 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     # Course Goals
     'lms.djangoapps.course_goals.apps.CourseGoalsConfig',
 
+    # Tagging
+    'openedx_tagging.core.tagging.apps.TaggingConfig',
+    'openedx.features.content_tagging',
```

Comment:
(This can come in a separate PR)

It would be good to add `openedx.features.content_tagging` to [`mypy.ini`](https://github.com/openedx/edx-platform/blob/e7469868207eb3ad614f428a8cbdf7a4b1f48de0/mypy.ini#L9-L14) so you get static type checking, and to [`setup.cfg` `isolated_apps`](https://github.com/openedx/edx-platform/blob/e7469868207eb3ad614f428a8cbdf7a4b1f48de0/setup.cfg#L168-L175) so you can make sure all other parts of the code are only importing from `content_tagging.api`. I think you can add `openedx_tagging.core.tagging` to `isolated_apps` as well to make sure we're only importing from `api`.

Reply:
Oo that's very cool.. done with e22f1db4a0

Reply:
cf https://github.com/openedx/modular-learning/issues/83